packages/@tinacms/cli/src/cmds/audit/audit.ts

```diff
@@ -121,6 +121,24 @@ export const auditDocuments = async (args: AuditArgs) => {
       query: documentQuery,
       variables: {},
     })
+
+    Object.keys(docResult.data.document._values)
+      .filter((fieldName) => {
+        return docResult.data.document._values[fieldName]?.type == 'root'
+      })
+      .forEach((fieldName) => {
+        const errorMessages = docResult.data.document._values[
+          fieldName
+        ].children
+          .filter((f) => f.type == 'invalid_markdown')
+          .map((f) => f.message)
+
+        errorMessages.forEach((errorMessage) => {
+          error = true
+          logger.error(chalk.red(errorMessage))
```

Comment:
perhaps this should be a `warn`. [Here's](https://github.com/tinacms/tinacms/blob/392d8870281ee4d081d39cdb37ad4235011b86a2/packages/%40tinacms/cli/src/cmds/audit/audit.ts#L62-L71) an example of that.